dockers/docker-orchagent/supervisord.conf

```diff
@@ -62,9 +74,8 @@ dependent_startup_wait_for=orchagent:running
 command=/usr/bin/swssconfig.sh
 priority=6
 autostart=false
-autorestart=unexpected
+autorestart=false
 startretries=0
-startsecs=0
```

Comment:
i am not sure why you want to change this, can you explain the reason?

Reply:
I don't run in a loop, I want supervisord to consider a start and execution sub-second to be success. I experimented with this, only workable solution, otherwise, we see failure since the process executes rather quickly.

Reply:
> I want supervisord to consider a start and execution sub-second to be success

If this is the case, then you would want to leave `startsecs=0`. Otherwise, the deafult value is `startsecs=1`, meaning the process needs to stay running for at least 1 second in order to be considered successful.

Reply:
I'll try again, but as my comments indicated, I experimented and this was the only config i could get working.